### PR TITLE
Sassy

### PR DESCRIPTION
**Pull request by dvicklund**
Added sass processing gulp task with watch

### CODE REVIEW

File: david_vicklund/gulpfile.js [hunk 1]
```diff
@@ -23,6 +28,29 @@ gulp.task('jshint:test', function() {
...
+    }))
+    .pipe(gulp.dest('test/client/'));
+});
+
+// CSS tasks
+gulp.task('minify:scss', function() {
+  gulp.src('app/scss/*.scss')
+    .pipe(sass())
+    .pipe(gulp.dest('app/css/'));
+  return gulp.src('app/css/**/*.css')
```
Comment (warmbowski) [inline david_vicklund/gulpfile.js:46]: Im not sure what this line is for.

File: david_vicklund/gulpfile.js [hunk 2]
```diff
@@ -23,6 +28,29 @@ gulp.task('jshint:test', function() {
...
+    .pipe(sourcemaps.init())
+    .pipe(concatCss('styles.min.css'))
+    .pipe(minifyCss())
+    .pipe(sourcemaps.write())
+    .pipe(gulp.dest('build/'));
+});
```
Comment (warmbowski) [inline david_vicklund/gulpfile.js:52]: It looks like you've combined the CSS gulp task with the SCSS gulp task.

File: david_vicklund/build/index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>The Food is Around Your Face</title>
+  <link rel="stylesheet" href="styles.min.css">
```
Comment (warmbowski) [inline david_vicklund/build/index.html:6]: this should probably be href=application.css

File: david_vicklund/gulpfile.js (hunk 2 above)
Comment (dvicklund) [inline david_vicklund/gulpfile.js:52]: Yeah, that was sort of a side effect of doing the scss assignment before
the vanilla CSS

**Pull request merged**